onepane/README.md

```diff
@@ -0,0 +1,78 @@
+# Onepane
+
+## Overview
+
+Onepane is a powerful addition to your existing Datadog, Cloud, and DevOps tools, providing seamless out-of-the-box integrations. It adopts an agentless approach and features automatic asset discovery, allowing you to get value in a matter of hours, not days or weeks. Onepane comprehends your unique landscape, generating a dynamic service map of your operational environment by harnessing data from various systems. This map assists your teams in swiftly resolving incidents and streamlines the audit process. Opt for Onepane to leverage additional ROI on your existing investments. 
+
+- Show changes and infrastructure events alongside errors to pinpoint potential root causes.
+- Enable root cause investigation by correlating incidents with change data and infrastructure data.
+
+## Setup
```

Comment:
OAuth must be kicked off from Datadog. Please include the exact steps a user will take within Datadog to set up the integration. For example, go to the Onepane integration on the integrations page, select install, click connect accounts, log in to your onepane account, .... etc

Reply:
The steps below do not walk through the OAuth flow. Please update the steps to walk through OAuth steps a user will take to install the integration as mentioned above. 

Users should not be creating api or app keys within their account as OAuth replaces the need for this.